prod-cockpit-rcar.yaml

```diff
@@ -5,6 +5,7 @@ variables:
   YOCTOS_WORK_DIR: "yocto"
   DOM0_BUILD_DIR: "build-dom0"
   DOMD_BUILD_DIR: "build-domd"
+  CLUSTER_BUILD_DIR: "cluster-wrapper-build"
```

Comment:
Not sure that this change should go in "meta-xt-prod-cockpit-rcar-qt5: add layer to support the qt5." commit